lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -108,11 +109,11 @@
     % endif
 
     <hr>
-    <p class="start-certificate-regeneration">
-        <h2>${_("Regenerate Certificates")}</h2>
+    <div class="start-certificate-regeneration">
+        <h3 class="hd hd-3">${_("Regenerate Certificates")}</h3>
         <form id="certificate-regenerating-form" method="post" action="${section_data['urls']['start_certificate_regeneration']}">
             <p class="under-heading">
-                ${_('To regenerate certificates for your course, choose the learners who will receive regenerated certificates and click <br/> Regenerate Certificates.')}
+                ${Text(_('To regenerate certificates for your course, choose the learners who will receive regenerated certificates and click Regenerate Certificates.'))}
```

Comment:
There's no need to use "Text" here because you aren't mixing together Text and HTML.